server/admin_management/tests/conftest.py

```diff
@@ -130,6 +140,126 @@ def test_rsa_key_missing():
     return new_key.exportKey("PEM")
 
 
+@pytest.fixture(scope="function")
+def setup_new_user(user_repo: UserRepository, db_pg_session: Session):
```

Comment:
I wonder if we could add the return type for this method? as well as the setup_new_fom_delegated_admin?